common/src/main/java/io/pravega/schemaregistry/common/HashUtil.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.common;
+
+import lombok.SneakyThrows;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class HashUtil {
+    private static final String SHA_256 = "SHA-256";
+
+    /**
+     * Computes a 256 bit hash of supplied bytes using sha-256 hash function.
+     *
+     * @param bytes bytes to compute hash of. 
+     * @return a 256 bit hash of the given bytes.
+     */
+    @SneakyThrows(NoSuchAlgorithmException.class)
+    public static BigInteger getFingerprint(byte[] bytes) {
+        MessageDigest md;
+        md = MessageDigest.getInstance(SHA_256);
+
+        return new BigInteger(md.digest(bytes));
```

Comment:
as long as we create the same hash each time and converting bigInteger to a byte array for serializing it and back from byte array on deserialization gives us same hash we are fine.

Reply:
For the usage you have mentioned, it wouldn't matter. But if you log it or use it in some other contexts this might be awkward (if not problematic) to work with. It may be useful to make it positive, although it's not necessary.